python_app/models.py

```
@@ -0,0 +1,64 @@
+"""Data models for home monitoring sensor readings."""
+
+import enum
+import math
+import statistics
+import uuid
+import socket
+import struct
+from typing import Any, Dict, List, Optional, NamedTuple, Set
```

Ruff (0.15.6)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)